code/modules/mob/living/carbon/human/species_types.dm

```diff
@@ -623,6 +623,11 @@
 						"<span class='userdanger'>You throw up on the floor!</span>")
 	..()
 
+/datum/species/fly/check_weakness(obj/item/weapon, mob/living/attacker)
```

Comment:
This is true, but you *can* apply damage and check for armor..or you can just make it bypass armor completely and do `take_overall_damage`, `adjustBruteLoss`, or the likes.